src/envo/envo-edit.owl

```diff
@@ -40125,6 +40127,37 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105010> "plastic pellet"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105010> <http://purl.obolibrary.org/obo/ENVO_01000776>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105014> (biodegradation)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://doi.org/10.1351%2FPAC-REC-10-12-04>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105014> "A biological process during which substance or chemical is broken down due to the action of cells."@en)
```

Comment:
>A biological process during which a material is broken down.

Did you confirm that this applies to non-organic materials?

Reply:
@pbuttigieg 

Yes. Even though biodegradation of inorganic compounds are not as common, they do happen. For example in [this paper](https://pubs.acs.org/doi/full/10.1021/acs.bioconjchem.9b00699) they summarize inorganic nanoparticles (e.g. silica) that are biodegradable and being used in cancer treatment.